turbopack/crates/turbopack-ecmascript/src/utils.rs

```diff
@@ -78,6 +78,23 @@ pub fn module_id_to_lit(module_id: &ModuleId) -> Expr {
     })
 }
 
+pub struct StringifyModuleId<'a>(pub &'a ModuleId);
+
+impl std::fmt::Display for StringifyModuleId<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self.0 {
+            ModuleId::Number(n) => {
+                if *n <= JS_MAX_SAFE_INTEGER {
+                    n.fmt(f)
+                } else {
+                    write!(f, "\"{n}\"")
```

Comment:
Why we would end up in this case? Shouldn't we avoid to have module ids that large in first place?

Reply:
Right, it shouldn't. I was copying the logic from `module_id_to_lit` here, which does the same.